### PR TITLE
Whispering in crit makes you succumb again

### LINKED ISSUE #21137: can't whisper in crit or say *deathgasp in crit to succumb
(WEB REPORT BY: whoru REMOTE: 206.221.180.138:7777)
I had clone damage at the time if it helps

### PR DESCRIPTION
**Pull request by ShiftyRail**
[bugfix]
Closes #21137 
This is very inelegant but I'm not sure if I can do it otherwise

### CODE REVIEW

**Comment by SonixApache** [comment]
did you check if the bustanuts still work as intended

File: code/modules/mob/living/living.dm [hunk 1]
```diff
@@ -178,10 +178,15 @@
 
 /mob/living/verb/succumb()
 	set hidden = 1
+	succumb_proc(0, 1)
+
+/mob/living/verb/succumb_proc(var/gibbed as num, var/deathgasping as num)
```
Comment (Intigracy) [inline code/modules/mob/living/living.dm]: don't make this a verb

**Comment by ShiftyRail** [comment]
@SonixApache Can you clarify ?

**Comment by ghost** [comment]
Dying with Bustanut in you used to make you whisper "i told u i was hardcore..", which is probably what he's thinking of, but now I've taken a look at it it looks like that was removed in #19648 along with most of the rest unique deathgasps.

**Comment by ShiftyRail** [comment]
Oh right, may as well be a good time to re-implement them.

**Comment by ShiftyRail** [comment]
There

**Pull request merged**